packages/form-control/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "@open-wc/form-control",
```

Comment:
Important pre-ship question, do you feel there will be other pieces of "form participation functionality" that you would want to add later in a way that would benefit from naming this `@open-wc/form-participation` with multiple features to this process in the sub-directories of `src`?

Reply:
Form participation makes a lot of sense, I'll make that change in the next push.